### PR TITLE
Feature/about

### PR DESCRIPTION
**Pull request by mumina-abdo**
# Engineers Checklist

## What this PR does?
- List of updates for features, enhancements, or bugfixes

## How can it be tested?
- Please provide step-by-step guide on how to test
- If web or mobile, screenshots will be very helpful in addition

# Where is it documented?
- List JIRA tickets

## Backend Updates
- [ ] Updated Postman Documentation
	- [ ] Requests
	- [ ] Examples
	- [ ] Tests
- [ ] Updated Tasks on JIRA
- [ ] Tested API updates in `dev` environment
- [ ] CI Checks Passed

### CODE REVIEW

**Review by Gladwell-Wanjiku — DISMISSED** [review]

**Review by Maureen-Njango — APPROVED** [review]

**Review by Agnes-Ajema — DISMISSED** [review]

**Review by Fridah-Wothay — APPROVED** [review]

File: real/src/About/index.css [hunk 1]
```diff
@@ -0,0 +1,330 @@
+body{
```
Comment (hunterachieng) [inline real/src/About/index.css:1]: Wont this affect the whole website?

File: real/src/About/index.css [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+
+@media screen and (min-width:350px) and (max-width:360px){
+
+    body{
+       background-color: rgb(223, 209, 209);
+    }
+    
+    .about-container{
+        font-size: 15px;
+        /* text-align: center; */
```
Comment (hunterachieng) [inline real/src/About/index.css:80]: Remove commented code

File: real/src/App.js [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+import About from './About';
+import Team from  './Team';
+
+function App() {
+  return (
+    <div className="App">
+      <Product/>
+      <About/>
+      <Team/>
+=======
```
Comment (hunterachieng) [inline real/src/App.js:14]: What is happening here?

File: real/src/Product/index.css [hunk 4]
```diff
@@ -3,11 +3,19 @@
     gap: 5%;
     justify-content: center;
     margin-top: 6%;
+<<<<<<< HEAD
```
Comment (hunterachieng) [inline real/src/Product/index.css:6]: What is happening here?

File: real/src/Product/index.css [hunk 5]
```diff
@@ -27,6 +35,13 @@ top: 25px;
 
 }
 
+<<<<<<< HEAD
+.largecontainer{
+    margin-bottom: 12%;
```
Comment (hunterachieng) [inline real/src/Product/index.css:40]: What is happening here?

File: real/src/Product/index.css [hunk 6]
```diff
@@ -87,15 +186,31 @@ top: 25px;
 @media (min-width: 1020px) and (max-width: 1280px){
     .images{
         display: flex;
+<<<<<<< HEAD
+        /* gap: 10%; */
+=======
```
Comment (hunterachieng) [inline real/src/Product/index.css:191]: What is happening here?

File: real/src/Product/index.css [hunk 7]
```diff
@@ -87,15 +186,31 @@ top: 25px;
...
+        margin-right: 5%;
+        margin-left: 9%;
+        
+        
+=======
         width: 400px;
         height: 380px;
         /* margin-top: 5%; */
         border-radius: 10%;
+>>>>>>> b0f50eb61171a0456826185bdcd3c456746dfc6c
```
Comment (hunterachieng) [inline real/src/Product/index.css:213]: What is happening here?

File: real/src/Product/index.css [hunk 8]
```diff
@@ -112,15 +231,28 @@ top: 25px;
...
+        /* margin-top: 5%; */
+        border-radius: 10%;
+        margin-right: 5%;
+        margin-left: 5%;
+=======
         width: 300px;
         height: 280px;
         /* margin-top: 5%; */
         border-radius: 10%;
+>>>>>>> b0f50eb61171a0456826185bdcd3c456746dfc6c
```
Comment (hunterachieng) [inline real/src/Product/index.css:255]: What is happening here?

File: real/src/Product/index.js [hunk 9]
```diff
@@ -1,10 +1,16 @@
...
+const Product = () =>{
+    return(
+     <div className="largecontainer">
+=======
 
 // const images () =
 const Product = () =>{
     return(
      <div>
+>>>>>>> b0f50eb61171a0456826185bdcd3c456746dfc6c
```
Comment (hunterachieng) [inline real/src/Product/index.js:13]: What is happening here?

File: real/src/Product/index.css (hunk 5 above)
Comment (mumina-abdo) [reply real/src/Product/index.css:40]: I will work on it now

File: real/src/About/index.css (hunk 1 above)
Comment (mumina-abdo) [reply real/src/About/index.css:1]: It will

File: real/src/About/index.css (hunk 2 above)
Comment (mumina-abdo) [reply real/src/About/index.css:80]: I will

File: real/src/Product/index.css (hunk 4 above)
Comment (mumina-abdo) [reply real/src/Product/index.css:6]: Am on it

**Review by hunterachieng — DISMISSED** [review]

**Review by Agnes-Ajema — DISMISSED** [review]

**Pull request closed without merging**